### ISSUE TITLE
Location is not available / accessible after some time

### ISSUE THREAD

**Issue by octylFractal** [maharmstone/btrfs#297]
After a little bit of time, not exactly sure how much but about a few hours, I end up with this message when trying to open the drive in File Explorer:
![message](https://url.octyl.net/bsy0cpzi6ikx-4xn--w1)

If you need any logs, let me know. I'm using 1.7.4.

P.S. The recent release has otherwise been a big improvement. Thank you for your hard work!

**Comment by maharmstone** [comment]
I've not seen this before. If you log off and log back in (rather than rebooting), is it still inaccessible?

**Comment by octylFractal** [comment]
No, a log out and in fixes the issue. It seems that kills a process labelled as `dllhost.exe` in Process Explorer, which is holding `H:\` specifically.

**Comment by MakiseKurisu** [comment]
I'm experencing this issue as well. During this time the root of drive is inaccessible, but individual files inside said drive seems to work fine, as my music player is able to keep playing. Reboot fixes this issue. I'll try log out and use Process Explorer to see what's going on.

**Comment by MakiseKurisu** [comment]
In my case the handle is held by a 3rd party tool called Everything. Once I closed the handle in Process Explorer I'm able to access my drive again.

**Comment by AWiederin** [comment]
I have had this sort of issue as well. Turning off Everything seemed to reduce the frequency a lot, but still it sometimes seems as if Windows (explorer, then task manager, etc.) stalls while waiting to access something.

**Comment by maharmstone** [comment]
I can't say for certain, as I couldn't reproduce the issue, but https://github.com/maharmstone/btrfs/commit/0c910f3bec52e2b9484c9deeb50b9968260db2f3 might improve matters.

**Comment by octylFractal** [comment]
I would give it a try, but unfortunately I've never been able to get a version built from source to install properly on my computer. If you're able to provide a build or instructions for correctly assembling the driver, I am willing to try it out.

**Comment by Trevomader** [comment]
I also have this issue, happens mostly by using Edge Chromium, I have to close the process to be able to access the volume again. Handles are being held as shown by ProcessHacker.

**Comment by Trevomader** [comment]
Just installed a build from "master" including commit 0c910f3, but still getting locks out of the filesystem.
